upload-ami/src/upload_ami/smoke_test.py

```diff
@@ -80,9 +80,10 @@ def main() -> None:
     parser.add_argument("--image-id", required=True)
     parser.add_argument("--run-id", required=False)
     parser.add_argument("--cancel", action="store_true", required=False)
+    parser.add_argument("--instance-market-options", default='{"MarketType": "spot"}', required=False)
```

Comment:
I don't think this works.   CLI arguments can only be strings or bools.    Or does argparse automatically parse dicts too?